portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -25,6 +28,7 @@
 import javax.servlet.http.HttpServletResponse;
 
 
```

Comment:
probable don't need this many empty lines (linter usually yells at us when there's more than one empty line in a row)